unstructured/chunking/title.py

```diff
@@ -9,11 +10,80 @@
     Element,
     ElementMetadata,
     Table,
+    TableChunk,
     Text,
     Title,
 )
 
 
+def chunk_by_characters(
+    elements: List[Element],
+    num_characters: Optional[int] = 200,
+) -> List[Element]:
+    """Uses num_characters to return elements within a document in specific sizes.
+    Currently only applies to Table elements.
+     Parameters
+    ----------
+    elements
+        A list of unstructured elements. Usually the output of a partition functions.
+    num_characters
+        Splits elements text field so that a section matches the desired length."""
+
+    chunked_elements: List[Element] = []
+
+    for element in elements:
+        if isinstance(element, Table):
+            char_len = len(element.text)
+            html_table = element.to_dict().get("text_as_html")
+            if html_table:
+                char_len = len(html_table)
+            if char_len > num_characters:  # type: ignore
+                chunked_elements.extend(chunk_table_element(element, num_characters))
```

Comment:
here we are potentially mixing `Element` objects with `Chunk` objects; are those two types of objects compatible? Are there attributes that exist in one but not another?

Reply:
nevermind, I see element is essentially the parent